db/migrate/20180115215649_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name, null: false
+      t.string :email, null: false
+      t.string :password_digest
+      t.string :uid
```

Comment:
What is `:uid` for?

Reply:
I will use :uid with omniauth for Facebook, should I have waited to add this column?